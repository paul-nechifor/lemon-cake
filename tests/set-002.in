(set (set (dict) 'name' 'John') 1 2)
