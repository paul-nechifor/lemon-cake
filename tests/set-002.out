(dict 1 2 'name' 'John')
